src/channel-list-item.tsx

```diff
@@ -32,12 +34,12 @@ export class ChannelViewModel extends Model {
       .map((starred: any) => starred.value)
       .toProperty(this, 'starred');
 
-    this.when('model.dm_count', 'model.mention_count_display',
-      (dmCount, mentions) => (dmCount.value || 0) + (mentions.value || 0))
+    this.when('model.mention_count')
```

Comment:
We should really make `when` just always return the value, and make some other method for what `when` does now, any ideas for a name?